docs/blog/2021-12-09-log4j-zero-day.mdx

```diff
@@ -25,18 +25,18 @@ authors: [free, chris, forrest]
 
 _Originally Posted @ December 9th & Last Updated @ December 19th, 3:37pm PST_
 
-**Fixing Log4Shell? See Our [Updated Mitigation Guide](https://www.lunasec.io/docs/blog/log4j-zero-day-mitigation-guide)
-including our automated scanning tool**
+**Fixing Log4Shell? See Our [Updated Mitigation Guide](https://www.lunasec.io/docs/blog/log4j-zero-day-mitigation-guide),
+including our automated scanning tool.**
 
-**Also read: Our analysis of [CVE-2021-45046](https://www.lunasec.io/docs/blog/log4j-zero-day-update-on-cve-2021-45046) (a second log4j vulnerability)**
+**Also read: Our analysis of [CVE-2021-45046](https://www.lunasec.io/docs/blog/log4j-zero-day-update-on-cve-2021-45046) (a second log4j vulnerability).**
 
 ## What is it?
-On Thursday (December 9th), a 0-day exploit in the
-popular Java logging library `log4j` (version 2) was discovered that results in Remote Code Execution (RCE) by
+On Thursday, December 9th, a 0-day exploit in the
+popular Java logging library `log4j` (version 2) was discovered that resulted in Remote Code Execution (RCE), by
```

Comment:
present tense is correct here..cant explain why but..computer words

Reply:
Gotcha